2000_run.py

```diff
@@ -0,0 +1,15 @@
+
+# write a function that receives another function and other parameters
+# the function will calculate how long f takes to run with those parameters
+# and return the time it took to run
+
+
+def timer(f, *args):
+    import time
+    start = time.time()
+    f(*args)
```

Comment:
rename 'f' Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing